platforms/wiced/libraries/memfault/platform_reference_impl/memfault_platform_http_client.c

```diff
@@ -182,7 +182,7 @@ static wiced_result_t prv_connect_and_send_coredump_request(
                                    security, MEMFAULT_HTTP_CONNECT_TIMEOUT_MS));
 
   char content_length[12] = {0};
-  snprintf(content_length, sizeof(content_length), "%d", cd_info->total_size);
+  snprintf(content_length, sizeof(content_length), "%zu", cd_info->total_size);
```

Comment:
We'll have to double check on the platform. Not every embedded printf implementation supports `zu`. cc @chrisc11

Reply:
Another option would be to cast to `intptr_t` and use `PRIuPTR`, which would be large enough for any value `size_t` can take.